config/locales/en.yml

```diff
@@ -1232,187 +1232,187 @@ en:
           halt: "Train Stop"
-          junction: "Railway Junction"
-          level_crossing: "Level Crossing"
-          light_rail: "Light Rail"
-          miniature: "Miniature Rail"
+          junction: "Railroad Junction"
+          level_crossing: "Railroad Crossing"
+          light_rail: "Light Rail Track"
+          miniature: "Miniature Train Track"
           monorail: "Monorail"
-          narrow_gauge: "Narrow Gauge Railway"
-          platform: "Railway Platform"
-          preserved: "Preserved Railway"
-          proposed: "Proposed Railway"
-          rail: "Rail"
-          spur: "Railway Spur"
-          station: "Railway Station"
-          stop: "Railway Stop"
-          subway: "Subway"
+          narrow_gauge: "Narrow Gauge Track"
+          platform: "Train Platform"
+          preserved: "Preserved Train Track"
+          proposed: "Proposed Train Track"
+          rail: "Train Track"
+          spur: "Spur Track"
+          station: "Train Station"
+          stop: "Train Stopping Position"
+          subway: "Subway Track"
           subway_entrance: "Subway Entrance"
-          switch: "Railway Points"
-          tram: "Tramway"
-          tram_stop: "Tram Stop"
+          switch: "Railroad Switch"
+          tram: "Trolley Track"
+          tram_stop: "Trolley Stopping Position"
           turntable: "Turntable"
-          yard: "Railway Yard"
+          yard: "Rail Yard"
         shop:
-          agrarian: "Agrarian Shop"
-          alcohol: "Off License"
-          antiques: "Antiques"
-          appliance: "Appliances Shop"
-          art: "Art Shop"
-          baby_goods: "Baby Goods"
-          bag: "Bag Shop"
+          agrarian: "Farm Supply Shop"
+          alcohol: "Liquor Store"
+          antiques: "Antique Store"
+          appliance: "Appliances Store"
+          art: "Art Store"
+          baby_goods: "Baby Goods Store"
+          bag: "Bag Store"
           bakery: "Bakery"
-          bathroom_furnishing: "Bathroom Furnishing"
-          beauty: "Beauty Shop"
-          bed: "Bedding Products"
-          beverages: "Beverages Shop"
+          bathroom_furnishing: "Bathroom Furnishing Store"
+          beauty: "Beautician"
+          bed: "Bedding Store"
+          beverages: "Beverage Store"
           bicycle: "Bicycle Shop"
           bookmaker: "Bookmaker"
-          books: "Book Shop"
+          books: "Bookstore"
           boutique: "Boutique"
           butcher: "Butcher"
-          car: "Car Shop"
-          car_parts: "Car Parts"
-          car_repair: "Car Repair"
-          carpet: "Carpet Shop"
-          charity: "Charity Shop"
-          cheese: "Cheese Shop"
-          chemist: "Chemist"
-          chocolate: "Chocolate"
-          clothes: "Clothes Shop"
-          coffee: "Coffee Shop"
-          computer: "Computer Shop"
-          confectionery: "Confectionery Shop"
+          car: "Car Dealership"
+          car_parts: "Car Parts Store"
+          car_repair: "Car Repair Shop"
```

Comment:
Maybe this is regional, but it seems to me that "Auto" or "Automotive" is far more common than "Car" in the context of parts and repair.  i.e. auto parts store, auto mechanic, automotive repair.  On the other hand, "Car Dealership" definitely sounds right to me.  I don't commonly hear or read about auto dealerships or automotive dealerships.

Reply:
There are a bunch of possibilities here – even just “Mechanic”. The main thing is that `shop=car_parts` is a “store” while `shop=car_repair` is a “shop” (as in “workshop”).